packages/shared/src/graphql/users.ts

```diff
@@ -24,7 +24,7 @@ export const USER_STATS_QUERY = gql`
 
 export type UserReadingRank = { currentRank: number };
 export type UserReadingRankData = { userReadingRank: UserReadingRank };
-export type MostReadTag = { value: string; count: number };
+export type MostReadTag = { value: string; count: number; percentage?: number };
 export type UserTooltipContentData = {
   rank: UserReadingRank;
   tags: MostReadTag[];
```

Comment:
Should we not uniform this with the TopTag interface?

Reply:
They have different APIs that have different shapes. So I think we won't be able to unify them.